app/components/Speakers/SpeakerPreview.jsx

```diff
@@ -53,32 +53,38 @@ export class SpeakerPreview extends React.PureComponent {
     return <Icon className="speaker-picture" name="user" size="large" style={{color: 'grey'}}/>
   }
 
-  renderActions() {
+  renderActions(speaker) {
     return (
       <div className="quick-actions">
         {this.props.speaker.is_user_defined &&
           <ReputationGuard requiredRep={MIN_REPUTATION_UPDATE_SPEAKER}>
             <ClickableIcon
+              href="#"
               name="pencil"
               size="action-size"
-              title={this.props.t('main:actions.edit')}
+              title={`${this.props.t('main:actions.edit')} ${speaker.full_name}`}
```

Comment:
`${this.props.t('main:actions.edit')} ${speaker.full_name}` duplication de code, il faut extraire ça dans une variable